server/interpreter.js

```diff
@@ -3043,7 +3376,7 @@ Interpreter.prototype.installTypes = function() {
   };
 
   /**
-   * Server is an (owner, port, proto, (extra info)) tuple representing a
+   * Server is an (owner, port, proto, (extra info) tuple representing a
```

Comment:
I think that ) was needed.

Reply:
Indeed.  Not sure how that happened; fat fingers, probably!